src/vellum/workflows/references/environment_variable.py

```diff
@@ -8,7 +8,7 @@
 
 
 class EnvironmentVariableReference(BaseDescriptor[str]):
-    def __init__(self, *, name: str, default: Optional[str]):
+    def __init__(self, *, name: str, default: Optional[str] = ""):
```

Comment:
Actually sorry I misspoke. Let's use the default of `None` like `environment.py` does above